ocf_data_sampler/torch_datasets/process_and_combine.py

```diff
@@ -119,8 +125,9 @@ def process_and_combine_site_sample_dict(
             data_arrays.append((f"nwp-{provider}", da_nwp))
           
     if "sat" in dataset_dict:
-        # TODO add some satellite normalisation
+        # Satellite normalisation added
```

Comment:
thanks for this

Reply:
@felix-e-h-p this is picky but can we change this comment to just `# Standardise` or something. I don't think "Satellite normalisation added" makes sense out of context now the TODO is gone